static/src/javascripts/projects/common/modules/experiments/tests/contributions-membership-epic-cta-rest-of-world.js

```diff
@@ -31,63 +32,54 @@ define([
+             intersection
 ) {
 
 
     return function () {
 
-        this.id = 'ContributionsMembershipEpicCtaRestOfWorld';
-        this.start = '2016-11-07';
+        this.id = 'ContributionsMembershipEpicCtaRestOfWorldTwo';
+        this.start = '2016-11-08';
         this.expiry = '2016-11-11';
         this.author = 'Jonathan Rankin';
-        this.description = '1) Find optimal way to present contributions and membership asks in Epic component. 2) Test 3 different messages for the Epic';
+        this.description = 'Test 3 different CTA configurations for the Epic, not in the US';
         this.showForSensitive = true;
         this.audience = 1;
         this.audienceOffset = 0;
         this.successMeasure = 'Impressions to number of contributions/supporter signups';
         this.audienceCriteria = 'All readers who are not the US, who are reading about US politics OR the US election, as well as not reading a Brexit articles ';
         this.dataLinkNames = '';
-        this.idealOutcome = 'We learn the best way to present contributions and membership asks in Epic component, and we lean what the most effective of the 3 messages is';
+        this.idealOutcome = 'We learn the best way to present contributions and membership asks in Epic component';
         this.canRun = function () {
+            var whitelistedKeywordIds = ['us-news/us-elections-2016', 'us-news/us-politics'];
+
+            var hasKeywordsMatch = function() {
+                if ('keywordIds' in config.page) {
+                    var pageKeywords = config.page.keywordIds.split(',');
+                    return intersection(whitelistedKeywordIds, pageKeywords).length > 0;
+                } else return false;
+            };
+
+
             var userHasNeverContributed = !cookies.get('gu.contributions.contrib-timestamp');
-            if('keywordIds' in config.page && 'nonKeywordTagIds' in config.page) {
-                var worksWellWithPageTemplate = (config.page.contentType === 'Article'); // may render badly on other types
-                var keywords = config.page.keywordIds.split(',');
-                var nonKeywordTagIds = config.page.nonKeywordTagIds.split(',');
-                var isAboutBrexit = (keywords.indexOf('politics/eu-referendum') !== -1) && (nonKeywordTagIds.indexOf('tone/news') !== -1);
-                var isMinuteArticle = ('isMinuteArticle' in config.page && config.page.isMinuteArticle);
-                var isAboutUsElectionOrUsPolitics = (keywords.indexOf('us-news/us-elections-2016') !== -1) || (nonKeywordTagIds.indexOf('us-news/us-politics') !== -1);
-                return !isMinuteArticle && !isAboutBrexit && isAboutUsElectionOrUsPolitics && userHasNeverContributed && commercialFeatures.canReasonablyAskForMoney && worksWellWithPageTemplate;
-            } else {
-                return false;
-            }
+            var isMinuteArticle = ('isMinuteArticle' in config.page && config.page.isMinuteArticle);
```

Comment:
Does this add much versus `var isMinuteArticle = config.page.isMinuteArticle`, which I think would perform the same function?